gr-iio/grc/iio_device_sink.block.yml

```diff
@@ -48,9 +48,16 @@ parameters:
     dtype: string
     hide: ${( 'part' )}
 
+-   id: ch_type
+    label: Channel type
+    dtype: enum
+    options: [complex, float, int, short, byte]
```

Comment:
`float` and `int` have the same size. Are both types actually available once the signature propagates down to the iio code? Same for the source block.

Reply:
No, just int .. I'm not aware of any devices that provide data in floating point format directly.
On that note, is there any integer 64 bit alternative available in the companion to replace complex with that ?

I'll push the changes tomorrow when I'm in the office.

Reply:
GRC defines "Complex Integer 64". Types can be passed to a block constructor explicitly (e.g, as an enum, as a varant of the block type), or by size. A lot of blocks infer type from size because the flowgraph validates connections by matching sizes. This leads to some ambiguity in flowgraph checking, but GRC and blocks can know the exact type.

Reply:
I couldn't find a list of possible values for types in that can be set in the yml file. Is it possible to add that here ?

Reply:
There a couple lists in `grc/core/Constants.py`: `CORE_TYPES` and `PARAM_TYPE_NAMES`. I think the latter is what applies here, and there is no 64-bit int in the list. If there were a 64-bit int, it would have the same size as complex float, too.

Anyway, I was more worried about the ambiguity of `float` and `int`, so you'd remove `float`?

Reply:
Yes, I just pushed .. removed float.